packages/core/http/core-http-router-server-mocks/src/router.mock.ts

```diff
@@ -34,6 +35,7 @@ function createRouterMock({ routerPath = '' }: { routerPath?: string } = {}): Ro
     patch: jest.fn(),
     getRoutes: jest.fn(),
     handleLegacyErrors: jest.fn().mockImplementation((handler) => handler),
+    versioned: createVersionedRouterMock(),
```

Comment:
In the interest of reducing core domains packages footprint, I'd prefer we move the versioned router mock into the router server mocks now rather than piling on yet more tech debt.